### PR TITLE
#164524812 add get all received user mail feature

### PR DESCRIPTION
**Pull request by vincentayorinde**
### What does this PR do?
- This PR will get all received user mail feature

### Description of Task to be completed?
- create message data structure
- add test for this endpoint /api/v1/messages
- create method for get all mails
- add input validations for this endpoint

### How should this be manually tested?
- Clone the repository from github and change directory into the repository
-  Access the hosted api URL + api/v1/message route to access  api end point
-  Test functionality

### What are the relevant pivotal tracker stories?
 [164524812](https://www.pivotaltracker.com/story/show/164524812)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -9,6 +9,20 @@ const app = express();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+
+app.use((req, res, next) => {
```
Comment (tomiadebanjo) [inline app.js]: Hi @vincentayorinde Nice work. But why did you add line 13 to 24

Comment (vincentayorinde) [reply app.js]: Fixed. Thank you
